server/vulnerabilities/msrc/io/fs_test.go

```diff
@@ -0,0 +1,47 @@
+package msrc_io
```

Comment:
general comment: I find the package naming using underscores a bit awkward, why not `mrscthing` or just `thing`?

> By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. Err on the side of brevity, since everyone using your package will be typing that name. And don't worry about collisions a priori. The package name is only the default name for imports; it need not be unique across all source code, and in the rare case of a collision the importing package can choose a different name to use locally. In any case, confusion is rare because the file name in the import determines just which package is being used.
>
> Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64. 
> Source https://go.dev/doc/effective_go#names

Reply:
Thanks, that's an old Python habit of mine I guess. 

So would it make sense to just use `msrc` as the package name all around?

Reply:
>Thanks, that's an old Python habit of mine I guess.

ha! that makes sense, seems very pythonic

>So would it make sense to just use msrc as the package name all around?

it depends, if you put everything flat (no subfolders) inside `msrc`, then that sounds good, if, for example you have a file in `msrc/io/fs.go`, then the package name ideally would be just `io`.

collisions are handled by the "importer", but if you want to give this package a more unique name, for example `msrcio`, then use `msrc/msrcio/fs.go`